package test;
import java.sql.SQLException;

import javax.naming.Context;
import javax.naming.InitialContext;
import javax.naming.NamingException;
import javax.sql.DataSource;

public class test {
	
	 
	try {
	    Context init = new InitialContext();
	           
	    DataSource ds = (DataSource)init.lookup("java:comp/env/jdbc/리소스명");
	    ds.getConnection();
	           
	} catch (NamingException e) {
	    e.printStackTrace();
	} catch (SQLException e1) {
	    e.printStackTrace();
	}

}
